addons/escoria-core/game/core-scripts/esc/commands/walk.gd

```diff
@@ -18,7 +19,7 @@ class_name WalkCommand
 func configure() -> ESCCommandArgumentDescriptor:
 	return ESCCommandArgumentDescriptor.new(
 		2,
-		[TYPE_STRING, TYPE_STRING, TYPE_INT],
+		[TYPE_STRING, TYPE_STRING, TYPE_BOOL],
 		[null, null, null]
```

Comment:
I think we should set the third default value to "false" instead of null.

Reply:
Done